emergency_stop: [ {type: 'button', id: 2} ]
emergency_release: [ {type: 'button', id: 3} ]
translate_left_right: [ {type: 'axis', id: 0} ]
translate_forward_backward: [ {type: 'axis', id: 1, invert: True} ]
translate_up_down: [ 
    {type: 'button', id: 6, on_value: -1},
    {type: 'button', id: 4}
]
rotate_x: [ {type: 'axis', id: 2, invert: True} ]
rotate_y: [ {type: 'axis', id: 3} ]
rotate_wrist: [ 
    {type: 'button', id: 7, on_value: -1},
    {type: 'button', id: 5}
]
fingers_open_close: [
    {type: 'axis', id: 5},
    {type: 'button', id: 1, on_value: -1},
    {type: 'button', id: 0}
]
home_arm: [ {type: 'button', id: 9} ]
dead_man: [ ]

description: |
    # JOYSTICK CONFIG FOR MODEL LOGITECH F710 (DirectInput mode)
    # ==========================================================
    #
    # Thanks to RIVeR-Lab/wpi_jaco for base of this image
    #
    #                Controls                
    #     up/down                 rotate wrist
    #    ________                 __________    
    #   /fin.open\_______________/          \   
    #  |   _| |_    < >  < home >     (4)    |  
    #  |  |_   _|   < >    < >     (1)   (3) |  
    #  |    |_|    ___       ___      (2)    |  
    #  | fin.close/   \     /   \            |  
    #  |          \___/     \___/            |  
    #  |       x/y trans  rotate x/y         |  
    #  |        _______/---\_______          |  
    #  |       |                    \        |  
    #   \      /                     \      /   
    #    \____/                       \____/    
    #                                        
    # Buttons:                               
    #   (1) Fingers open
    #   (2) Fingers close
    #   (3) Emergency stop                    
    #   (4) Emergency release
